docs/pages/database-access/guides/rds.mdx

```diff
@@ -36,8 +36,14 @@ This guide will help you to:
 
 - AWS account with RDS and Aurora databases and permissions to create and attach
   IAM policies.
-- A host, e.g., an EC2 instance, where you will run the Teleport Database
-  Service.
```

Comment:
we shouldn't remove this snippet, since we don't have this point after the warning admonition like we do on master:
>- A Linux host or Amazon Elastic Kubernetes Service cluster where you will run
  the Teleport Database Service, which proxies connections to your RDS
  databases.

Reply:
good catch!. bad merge, will fix.